lms/templates/ccx/enrollment.html

```diff
@@ -45,7 +46,7 @@
   </form>
 </div>
 
-<div class="member-lists-management" style="float:left;width:50%">
+<div class="member-lists-management" style="float:left;width:50%" aria-live="polite">
```

Comment:
@amir-qayyum-khan what kind of information in this div do you expect to change?  I only ask because putting an aria-live region on such a large area can make the page unnecessarily chatty.  Sometimes its necessary to limit the scope with the aria-relevant attribute.